frontend/assets/stylesheets/components/_packages.scss

```diff
@@ -362,6 +377,23 @@
     }
 }
 
+.package-promo--current {
+    background-color: $c-neutral5;
+}
+
+.package-promo--stretch {
+    @include mq(desktop) {
+        min-height: rem(gs-height(6.5));
```

Comment:
Does this number actually have a link to the grid system or is it just co-incidence? Might as well just hardcode it if it's magic. If not, ignore me!